mobile_files/package.json.orig

```diff
@@ -6,6 +6,7 @@
     "start": "node node_modules/react-native/local-cli/cli.js start"
   },
   "dependencies": {
+    "@react-native-community/netinfo": "^3",
```

Comment:
remove the ^

Reply:
Having the ^ here allows upgrading to a newer 3.x release on-demand by running `yarn update @react-native-community/netinfo`. Upgrades will not happen otherwise because we normally pass `--frozen-lockfile` to `yarn install`. I'm not saying what you suggest is wrong, but I'd like to understand the scenario we're trying to guard against.